### DISCUSSION TITLE
Using postee to send detection info

### DISCUSSION THREAD

**Discussion by simar7** [aquasecurity/tracee#590] [Development]
Can we leverage [postee](https://github.com/aquasecurity/postee) to send event info to other platforms as follows?

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/1254783/109737416-e39ee500-7b7a-11eb-8d45-7796d9c3e1f7.png">

**Comment by itaysk** [comment]
thanks for opening this, we have discussed it in the past which led to https://github.com/aquasecurity/tracee/issues/539. with that, people can choose which ever webhook server they want to use. 

if we'd want to replace falcosidekick as the default webhook server (TBD why), we'll need to map the significant gaps (not talking about supported integrations, but more fundamental charicteristics). as first impression, postee is very container scanning oriented, has some aqua specific flow and we'll need to see if that isn't too much to ship in the tracee container.

(as a side note, we should rethink if the webhook server should be bundled in the tracee container given the ultimate goal is k8s where it can be a sidecar)
